citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -135,7 +131,7 @@ protected void handleExportRequest(HttpServletRequest req, HttpServletResponse r
 			Reference ref, String format, String subtype) 
 			throws EntityNotDefinedException, EntityAccessOverloadException, EntityPermissionException 
 	{
-		if(! ContentHostingService.allowGetResource(req.getParameter("resourceId")))
+		if(! ContentHostingService.allowGetResource(req.getParameter("contentCollectionId")))
```

Comment:
Wasn't this in #11 ?

Why is this the right fix now, you're doing a permission check on one variable, and then exporting the content of another?